### ISSUE TITLE
Parser Exception for extra semicolon has no position information

### ISSUE THREAD

**Issue by leuschel** [hhu-stups/prob-issues#141]
If you have something like this
```
        IF value >0 THEN
           number_of_wins(state) := number_of_wins(state) + 1;
        ELSE
           number_of_losses(state) := number_of_losses(state) + 1;
        END
 ```
the BParser will only return this message
```
! An error occurred (source: parse_error) !
! Invalid semicolon after last substitution (before END)
```
without any line or column number.
This can obviously be problematic when the file is large.

The message seems to be generated in de.be4.classicalb.core.parser.analysis.checking in SemiColonCheck.java
```
		@Override
		public void caseAInvalidSubstitution(final AInvalidSubstitution node) {
			exceptions.add(new CheckException("Invalid semicolon after last substitution (before END)", node.getSemicolon()));
		}
```

**Comment by leuschel** [comment]
The issue seems to arise when there are two semicolons too many for an IF-THEN-ELSE
```
MACHINE SemicolonError
// There is no position for error message when you have two semicolons too many
OPERATIONS
  res <-- Operation1 = BEGIN
    IF 2>1 THEN
      res := TRUE;  // <---- ERROR
    ELSE
      res := FALSE; // <--- ERROR
    END
  END
END
```
With just one semicolon too much, we get a position information.

**Comment by leuschel** [comment]
The parsercall.pl module in prob_prolog did not really support compound_exception terms
coming from the parser.
Fixed in 763e3481737e0876e7fe0345872ada4a1074d1a9.
